randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -869,7 +876,9 @@ Extra - asset_id: collision_camera_012
   > Underwater Ledge Left
       After Ferenia - db_reg_b2_001
   > Event - Blob from above (db_reg_b2_001)
-      Wave Beam
+      Any of the following:
+          Wave Beam
+          Pseudo-Wave Beam (Beginner) and Shoot Diffusion Beam
```

Comment:
As Izzy has spoken, make this Advanced. Also, this specific pseudo needs Morph Ball to perform